app/graphql/resolvers/get_featured_articles_by_section_id.rb

```diff
@@ -10,23 +10,27 @@ class Resolvers::GetFeaturedArticlesBySectionID < GraphQL::Function
   # _ctx - is the GraphQL context (which would be discussed later)
   def call(_obj, args, _ctx)
     primary_article =
-      Article
+      select_published(
+        Article
         .joins('JOIN media ON articles.id = media.article_id')
         .joins('JOIN sections ON articles.section_id = sections.id')
         .where("sections.id = #{args['section_id']}")
         .order("articles.rank + 3 * sections.rank + 12 * articles.issue"\
                " + 192 * articles.volume DESC")
+      )
         .first
 
     secondary_articles =
+    select_published(
       Article
         .joins('JOIN sections ON articles.section_id = sections.id')
         .where("sections.id = #{args['section_id']} AND articles.id != #{primary_article.id}")
         .order("articles.rank + 3 * sections.rank + 12 * articles.issue"\
                " + 192 * articles.volume DESC")
+    )
 
     if primary_article.nil?
-      return secondary_articles.first(3)
+      return select_published(secondary_articles).first(3)
```

Comment:
Seems like you're calling `select_published` twice

Reply:
Yeah, that should be fixed.